samples/tab-approval/csharp/README.md

```diff
@@ -0,0 +1,139 @@
+﻿# Tab request approval with activity feed notification.
+
+This sample shows a feature where:
+1)Requester: Can request for any task to the manager by sending activity feed notification and can see his request status.
+2)Manager : Can see the pending request by user on the click of activity feed notification and can approve or reject the request.
```

Comment:
Can see the pending approval request raised by user on the click of activity feed notification and can approve or reject the request.

Reply:
updated